crates/hc_sandbox/src/save.rs

```diff
@@ -4,85 +4,168 @@
 //! to / from a `.hc` file.
 //! This is very much WIP and subject to change.
 
+use std::collections::HashSet;
 use std::path::{Path, PathBuf};
 use std::sync::OnceLock;
 
-use anyhow::Context;
+use crate::cmds::Existing;
 use holochain_conductor_api::conductor::paths::{ConfigFilePath, ConfigRootPath};
 
-/// Save all sandboxes to the `.hc` file in the `hc_dir` directory.
-pub fn save(mut hc_dir: PathBuf, paths: Vec<ConfigRootPath>) -> anyhow::Result<()> {
+/// Save all sandbox paths to the `.hc` file in the `hc_dir` directory.
+pub fn save(hc_dir: PathBuf, paths: Vec<ConfigRootPath>) -> std::io::Result<()> {
```

Comment:
Removed unnecessary `mut`  and changed return value to `std::io::Result` as `anyhow` is not recommended in library code.